docs/guide/testing-teal.md

```diff
@@ -4,60 +4,110 @@ layout: splash
-- [Parser](../packages/runtime/src/parser): Reads TEAL code and converts it to a list of opcodes which are executable by the interpreter. If any opcode/data in teal code is invalid, parser will throw an error.
-- [Interpreter](../packages/runtime/src/interpreter): Executes the list of opcodes returned by parser and updates stack after each execution. At the end of execution, if the stack contains a single non-zero uint64 element then the teal code is approved, and transaction can be executed.
 
 ## Block Rounds/Height
-In Algorand blockchain, the time is divided into rounds. Specifically, one round represents a a time to propose, validate and confirm a block.
+
+In Algorand blockchain, transaction processing divided into rounds. At each round blockchain creates a block with transactions which updates the state. All transactions in the same block has the same transaction time and block height.
 In Alogrand Builder Runtime, we don't have blocks. All transactions are processed immediately.
-However, we keep the notion of rounds and timestamps because it is needed for transaction and smart contract validation.
+However, we keep the notion of rounds and timestamps because it is needed for transaction and smart contract processing.
 
-The default Runtime block round is set to `2` and timestamp to `1`. Runtime doesn't change the block round or timestamp - it's up to the user and the test flow to manage the block. To change the block round and timestamp we need to call `runtime.setRoundAndTimestamp(round, timestamp)`. We can retrieve the current block round by using `runtime.getRound()` function and current timestamp by using `runtime.getTimestamp()`. <br />
+The default Runtime block round is set to `2` and timestamp to `1`. Runtime doesn't change the block round or timestamp - it's up to the user to set in when designing a test flow. To change the block round and timestamp we need to call `runtime.setRoundAndTimestamp(round, timestamp)`. We can retrieve the current block round and timestamp using `runtime.getRound()` and `runtime.getTimestamp()` respectively. <br />
 Example:
 
     runtime.setRoundAndTimestamp(5, 10); // set current block round to 5 and timestamp to 10
 
 This means that current block round is set to 5 and transaction will pass only if its' first valid round is less or equal 5 and the last valid round is greater than 5.
-Note: Block round and timestamp remains same until user changes it by calling `runtime.setRoundAndTimestamp(round, timestamp)`.
+Note: Block round and timestamp remains same until user will change it again.
+
+
+## Project Structure
+
+```
+algob project:
+├── assets
+│   ├── TEAL files
+│   ├── PyTEAL files
+├── scripts
+│   ├── deploy.ts
+│   ├── run.ts
+├── test
+│   ├── JS test files
+│   ├── .mocharc.json (optional)
+├── package.json
+```
+
+All our test files should be stored in `test` directory. You can import `algob` library as well as `scripts`.
```

Comment:
Could you please explain "You can import `algob` library as well as `scripts`" ? It doesn't sound very clear.

Reply:
I mean that in tests you can import `algob` as a library and functions from scripts.

Reply:
ok